dotcom-rendering/src/web/components/ArticleBody.tsx

```diff
@@ -48,6 +49,7 @@ type Props = {
 	tableOfContents?: TableOfContentsItem[];
 	lang?: string;
 	isRightToLeftLang?: boolean;
+	renderingTarget?: RenderingTarget;
```

Comment:
This component is consumed by many layouts, so I've made this prop optional so we can incrementally update layouts as we migrate them.